org.adempiere.base/src/org/compiere/print/ReportEngine.java

```diff
@@ -928,6 +930,29 @@ else if (colObj instanceof InstanceAttributeColumn)
 							tr.addElement(td);
 							Object obj = instanceAttributeColumn != null ? instanceAttributeColumn.getPrintDataElement(row)
 									: m_printData.getNodeByPrintFormatItemId(item.getAD_PrintFormatItem_ID());
+							//set style
```

Comment:
It looks strange to insert this block of code here. Should be before or after tr.addElement(td) above.